src/app/package.json

```diff
@@ -11,6 +11,7 @@
   "main": "main.js",
   "dependencies": {
     "@logtrine/logtrine": "latest",
+    "exif-js": "^2.3.0",
```

Comment:
Installing new modules requires to deploy a new desktop client, the current 6.1.7 client does not have this module. Furthermore electron is encouraging the usage of interprocess communication, that means `require()` modules directly in the browser is strongly discouraged and therefore may lead to migration issues someday.

I recommend using the browser version of [exif-js](https://cdn.jsdelivr.net/npm/exif-js) instead. Place it alongside the other [browser libraries](https://github.com/manga-download/hakuneko/tree/master/src/web/js) and add a reference to the [index](https://github.com/manga-download/hakuneko/blob/101a97a29b6b11c88980f53a7d6e95326d73ffab/src/web/index.html#L8).